src/sortStudents.ts

```diff
@@ -1,16 +1,72 @@
+/* eslint-disable implicit-arrow-linebreak */
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: Array<number>,
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+const avgGrades = (v: Array<number>): number => {
+  return v.length === 0
+    ? 0
+    : v.reduce((acc: number, grade: number) => acc + grade, 0) / v.length;
```

Comment:
```suggestion
    : v.reduce((acc grade) => acc + grade, 0) / v.length;
```
reduce is smart enough to infer types from passed array, you don't have to type them manually 😉